.zenodo.json

```diff
@@ -111,10 +111,16 @@
       "id": "GPL-3.0"
     },
     "references": [
-      "M. Vranic et al. (2015). Classical radiation reaction in particle-in-cell simulations. DOI:10.1016/j.cpc.2016.04.002",
-      "A. Gonoskov et al. (2015). Extended particle-in-cell schemes for physics in ultrastrong laser fields: Review and developments. DOI:10.1103/PhysRevE.92.023305",
       "L. V. Keldysh (1965). Ionization in the field of a strong electromagnetic wave.",
       "D. Bauer and P. Mulser (1999). Exact field ionization rates in the barrier-suppression regime from numerical time-dependent Schr\u00f6dinger-equation calculations. DOI:10.1103/PhysRevA.59.569",
-      "A. Huebl et al. (2015). openPMD 1.0.0 - A meta data standard for particle and mesh based data. DOI:10.5281/zenodo.33624"
+      "R. Pausch et al. (2014). How to test and verify radiation diagnostics simulations within particle-in-cell frameworks. DOI:10.1016/j.nima.2013.10.073",
+      "A. Huebl (2014). Injection Control for Electrons in Laser-Driven Plasma Wakes on the Femtosecond Time Scale. DOI:10.5281/zenodo.15924",
+      "M. Vranic et al. (2015). Classical radiation reaction in particle-in-cell simulations. DOI:10.1016/j.cpc.2016.04.002",
```

Comment:
This has been published in 2016

Reply:
Thus if you time ordering the publications, it should go after the openPMD standard.